src/core/physics/IPhysicsWorld.js

```diff
@@ -0,0 +1,171 @@
+
+define(["lib/classy",
+        "lib/three",
+        "core/physics/PhysicsRaycastResult",
+        "core/physics/CollisionInfo",
+        "core/framework/TundraSDK",
+        "core/framework/TundraLogging"
+    ], function(Class, THREE, PhysicsRaycastResult, CollisionInfo, TundraSDK, TundraLogging) {
+
+/**
+    Interface class for physics world.
+    
+    @class IPhysicsWorld
+    @constructor
+*/
+var IPhysicsWorld = Class.$extend(
```

Comment:
I think we could move this interface class, `PhysicsRaycastResult` and `CollisionInfo` into `core/physics/*. As these don't seem to include ammo.js directly (which is good ofc). The idea being we can have other implementations later without changing any app code (assuming they use the interface stuff only).

This would be similar to what renderer does https://github.com/realXtend/WebTundra/tree/dev2/src/core/renderer